### ISSUE TITLE
Data loading script failing 

### ISSUE THREAD

**Issue by samir-ware** [GoogleCloudPlatform/healthcare#27]
Hello 
Can you please help me out with the information of the folder name from where one need to execute following command ? 
`bazel run //datagen:data_gen -- --num=1 --output_path=demo_data.ndjson`

I tried running it on **"healthcare/fhir/immunizations_demo"**  folder but it failed with below error 

```
INFO: Invocation ID: 26092ad2-2ae0-49c9-ac2c-4988bf053007
ERROR: Skipping '//datagen:data_gen': no such package 'datagen': BUILD file not found on package path
WARNING: Target pattern parsing failed.
ERROR: no such package 'datagen': BUILD file not found on package path
INFO: Elapsed time: 0.070s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
```

**Comment by nikklassen** [comment]
A workaround for this is to run the command in healthcare/fhir/immunizations_demo/datagen.

**Comment by samir-ware** [comment]
@nikklassen Thank you for the reply although script is failing even in datagen folder. Please find below the stack trace 
```
samirs-MacBook-Pro:datagen unclesam$ bazel run //datagen:data_gen -- --num=1 --output_path=demo_data.ndjson
Starting local Bazel server and connecting to it...
INFO: Invocation ID: 957d2f1c-36f2-4ecb-ac3b-f40fc0d0e4b4
ERROR: Skipping '//datagen:data_gen': no such package 'datagen': BUILD file not found on package path
WARNING: Target pattern parsing failed.
ERROR: no such package 'datagen': BUILD file not found on package path
INFO: Elapsed time: 8.396s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
```

**Comment by samir-ware** [comment]
Running script in healthcare/fhir/immunizations_demo/ like 
`bazel run datagen:data_gen -- --num=1 --output_path=demo_data.ndjson`
results in following error 

```
samirs-MacBook-Pro:immunizations_demo unclesam$ bazel run datagen:data_gen -- --num=1 --output_path=demo_data.ndjson
Starting local Bazel server and connecting to it...
INFO: Invocation ID: 27c56358-1dea-42e5-ad9e-3a550bc360a4
INFO: Rule 'io_bazel_rules_python' modified arguments {"shallow_since": "1542396262 -0500"}
ERROR: /Users/unclesam/newDir10/healthcare/fhir/immunizations_demo/datagen/BUILD.bazel:3:1: no such package '@com_github_pborman_uuid//': The repository could not be resolved and referenced by '//fhir/immunizations_demo/datagen:data_gen'
ERROR: Analysis of target '//fhir/immunizations_demo/datagen:data_gen' failed; build aborted: no such package '@com_github_pborman_uuid//': The repository could not be resolved
INFO: Elapsed time: 29.288s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (10 packages loaded, 96 targets configured)
FAILED: Build did NOT complete successfully (10 packages loaded, 96 targets configured)
    currently loading: @io_bazel_rules_go//
```

**Comment by nikklassen** [comment]
Fixed with c8268338a986f531b7da2f4e5ecb939f247e6da8. The original command will work now. If you'd like to run bazel from another directory `bazel query ...` can help by showing you all available targets under the current directory.
